src/Lib.P2P/MultiAddressExtensions.cs

```diff
@@ -0,0 +1,147 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Threading;
+using System.Threading.Tasks;
+using MultiFormats;
+
+namespace Lib.P2P
+{
+    /// <summary>
+    ///   Extensions to <see cref="MultiAddress"/>.
+    /// </summary>
+    public static class MultiAddressExtensions
+    {
+        private static Dictionary<AddressFamily, string> _supportedDnsAddressFamilies =
+            new Dictionary<AddressFamily, string>();
+
+        private static MultiAddress _http = new MultiAddress("/tcp/80");
+        private static MultiAddress _https = new MultiAddress("/tcp/443");
+
+        static MultiAddressExtensions()
+        {
+            if (Socket.OSSupportsIPv4)
+                _supportedDnsAddressFamilies[AddressFamily.InterNetwork] = "/ip4/";
+            if (Socket.OSSupportsIPv6)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)